### ISSUE TITLE
Add support for feed to post

### ISSUE THREAD

**Issue by DanyCaissy** [Codeinwp/feedzy-rss-feeds#408]
Related to https://github.com/Codeinwp/feedzy-rss-feeds-pro/issues/287#issuecomment-599356835

We want to offer the feed to post feature to free users, so they can test it out and potentially pay for our pro version.

Let's enable it with these conditions:

* Limit of 10 posts per cycle
* Auto delete featured locked
* Custom fields locked
* All feed filters locked

Let's also ensure the user understands why these features are locked, at the top of the disabled fields we should have something like "These features are only enabled in the Pro version, click 'here' to learn more about the benefits of Pro"

With the upgrade link on the 'here': https://themeisle.com/plugins/feedzy-rss-feeds/upgrade/

**Comment by stefan-cotitosu** [comment]
@contactashish13 I have a suggestion on this one.

It will be helpful if we can show all the available magic tags for the post content field, in the third step of the import wizard, but make them unavailable, maybe show them like a placeholder or something like that.

Below is a screenshot of how I see it for the free version, also for the Personal one.
![suggestion-for-fremium-ftp](https://user-images.githubusercontent.com/26867648/77907981-53ce2800-7293-11ea-820b-81168c5c4af0.png)

**Comment by contactashish13** [comment]
@DanyCaissy maybe we can leave the names Agency and Business out as they tend to change and have to remain in sync with the website?

PS - I've locked the conversation so that no one outside of our group can post comments. Let me know if it stops anyone from commenting and I'll remove the lock.

**Comment by DanyCaissy** [comment]
@contactashish13 I think we could just lock the fields for that reason.

**Comment by rodica-andronache** [comment]
from what I've tested, this is working well. 
I think there are things to improve regarding the upsells, as they don't look great and it's hard to understand them in some cases, from my point of view.

For example, here is not clear that in the pro version you can have more options, other than 10. The upsell looks to override with the one for the option below.
![Apr-08-2020 12-23-24](https://user-images.githubusercontent.com/3266155/78767922-e2c5f900-7993-11ea-8f73-bb61ce1188e6.gif)

Here, as well, is not clear about which option we show the upsells for.
![Apr-08-2020 12-26-37](https://user-images.githubusercontent.com/3266155/78768409-8d3e1c00-7994-11ea-8bf7-fca9ea754e49.gif)

https://prnt.sc/rvafk0 - I think the link here should also be to https://themeisle.com/plugins/feedzy-rss-feeds/upgrade/

https://prnt.sc/rvaij7 - We should also update this text in the More Features tab, as the Feed to post is included in Free now

https://prnt.sc/rvahuj - the toggle is a bit off here

@DanyCaissy maybe you can help here with some solutions for the upsells, if you feel the same way about them

**Comment by contactashish13** [comment]
This isn't yet ready for testing. I'll assign to QA when it is :)

**Comment by rodica-andronache** [comment]
ok, sorry :)

**Comment by contactashish13** [comment]
PRs https://github.com/Codeinwp/feedzy-rss-feeds/pull/415 and https://github.com/Codeinwp/feedzy-rss-feeds-pro/pull/324 

I've incorporated the suggestions from https://github.com/Codeinwp/feedzy-rss-feeds/issues/408#issuecomment-610858702 as well

The following combinations are supported:
- new free/old pro
- new free/new pro

old free/new pro is not supported - this means https://github.com/Codeinwp/feedzy-rss-feeds-pro/pull/324 will fail until free is released.

**Comment by DanyCaissy** [comment]
@rodica-andronache Looks good functionality wise, ready for QA

**Comment by contactashish13** [comment]
@rodica-andronache this is now fixed.

**Comment by rodica-andronache** [comment]
@contactashish13 it works well now. You can fix the conflicts in #415 and Codeinwp/feedzy-rss-feeds-pro#324 so they can be merged

**Comment by contactashish13** [comment]
@rodica-andronache I've resolved the conflicts. 

I believe both of us overlooked the test for canonical URL corresponding to this setting. 

![image](https://user-images.githubusercontent.com/12953439/81902966-9e56fa00-95de-11ea-9544-986cc596ff7a.png)

I have now added support for that as well. Here is how to test it:

- import a post using feedzy
- uncheck the canonical url checkbox
- view the post on the front end. The canonical URL meta should either not exist or display the URL of the post
- check the canonical url checkbox
- view the post on the front end. The canonical URL meta should display the **original URL** of the feed item

The above can be tested using both Yoast and AIOSEO too.



The pro PR will fail till free is released.
